app/views/accounts/new.html.erb

```diff
@@ -1,12 +1,12 @@
 <h1 class="text-3xl font-semibold font-display"><%= t('.title')%></h1>
 
-<%= modal do %>
+<% content_for_modal = capture do %>
```

Comment:
We should update the modal helper to accept the args, and directly rather than bringing it's complexity into our view.

The modal helper is just a function at the end of the day so can accept any args we want:
```
<%= modal(controllers: "list-keyboard-navigation")
```

That said, if we take this approach, I also feel like it should accept args as if it were any regular tag builder helper. This would add a fair amount of complexity to the helper as we would need to figure out how to merge it's base properties and attributes with those we pass in here.

I think we can take an even simpler approach and just render a div inside the modal that wraps all the content and applies our list-keyboard-navigation controller.

Reply:
sure, will update the pr, thanks for the feedback!

Reply:
@JoshAntBrown updated, please take a look